app/models/local_jurisdiction.rb

```diff
@@ -0,0 +1,8 @@
+class LocalJurisdiction < ApplicationRecord
+  # Associations
+  has_many :permit_applications
+  has_many :review_managers, -> { where(role: User.roles[:review_manager]) }, class_name: "User"
+  has_many :reviewers, -> { where(role: User.roles[:reviewer]) }, class_name: "User"
+
```

Comment:
I see that Contact belong's to `local_jurisdiction`. Should there be a has many contacts relationship here?

Reply:
yes